boltdesignsystem/bolt
# DS-490: jest test image update

## Jira

[https://pegadigitalit.atlassian.net/browse/DS-490](https://pegadigitalit.atlassian.net/browse/DS-490)

## Summary

Standardize Jest Tests for Image

## Details

Standardize Jest Tests for Image. We're allowing for an easily repeatable test structure as well as improving the stability of the test being run on Travis, and making consistent formatting.

## How to test

Check out the code here, as well as the snapshots and make sure we're seeing what we want to see.
```
npx jest packages/components/bolt-image/tests/image.js
```
